tasks/cargo.yml

```diff
@@ -12,7 +12,7 @@
   with_items: "{{ cargo_result.results }}"
   changed_when: false
   failed_when:
-    - ("Add --force to overwrite" not in item.stderr) and
+    - ("use --force to override" not in item.stderr) and
```

Comment:
Maybe better to replace this with `(("use --force to override" not in item.stderr) or ("Add --force to overwrite" not in item.stderr)) and` so it stays backwards compatible